testsuite/single_node_performance.py

```diff
@@ -156,7 +156,7 @@ class RunGroupConfig:
     RunGroupConfig(expected_tps=1035, key=RunGroupKey("liquidity-pool-swap-stable"), included_in=Flow.CONTINUOUS),
     RunGroupConfig(expected_tps=4570, key=RunGroupKey("liquidity-pool-swap-stable", module_working_set_size=20), included_in=Flow.CONTINUOUS),
 
-    RunGroupConfig(expected_tps=27000, key=RunGroupKey("deserialize-u256"), included_in=Flow.CONTINUOUS),
+    RunGroupConfig(expected_tps=19250, key=RunGroupKey("deserialize-u256"), included_in=Flow.CONTINUOUS),
```

Comment:
@vusirikala you added this yesterday but looks like it consistently fail to meet your number..

Reply:
I put in the number based on my local execution. If the forge number is close to 19K, we can use that instead. Totally fine.

Reply:
for context - CONTINUOUS are not run on PRs, but on main. so having it off is not as urgent. 

but when adding a new workload, that hasn't been calibrated, you should use `waived=true`, and that will run them - but not have test fail - so you can remove it after recalibration. @vusirikala